feedback-module/src/components/Module.js

```diff
@@ -2,29 +2,33 @@ import React, { useEffect, useState, useRef } from "react";
 import { GridContainer, Grid, Form } from "@trussworks/react-uswds";
 import { useTranslation } from "react-i18next";
 
-import { SCREENS, INITIAL_SCREEN } from "../lib/constants";
+import { SCREENS, INITIAL_SCREEN, VALID_ENDPOINTS } from "../lib/constants";
```

Comment:
Should we change this variable name to "ENDPOINTS". I feel like VALID_ENDPOINTS is a little misleading because it almost seems like we are also storing invalid endpoints somewhere